stories/form/form-select/doc-element.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+
+export const ImageWrapper = styled.div`
+    display: flex;
+    border-radius: 4px;
+    border: 1px solid;
```

Comment:
is there no color? if there isnt a color, can just specify `border-width`

Reply:
found that minimally need to specify `border-style`

color will inherit the current color